src/core/icons/fileIcons.ts

```diff
@@ -3382,9 +3382,8 @@ export const fileIcons: FileIcons = {
       fileExtensions: ['cue'],
     },
     {
-      name: 'warp',
-      fileNames: ['warp.md'],
-      light: true,
```

Comment:
Hi @writedev! It seems like you accidentally replaced the `warp` file icon association with your new config in your latest merge.

That's why [CI is failing](https://github.com/material-extensions/vscode-material-icon-theme/actions/runs/18955589287/job/54131450081?pr=3243)

Would you mind fixing it?